### DISCUSSION TITLE
Welcome to hirob-em-embedded Discussions!

### DISCUSSION THREAD

**Discussion by husamettinarabaci** [Ubn-Jr/hirob-em-embedded#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by baburakbas** [comment]
Hello everyone!
I am Babür and I am working in the embedded software team in the Hirob project. I'm studying in Electrical Electronic so i have experience in this field. We are responsible for mechanical functionalities to hi-rob as like as make robot move, turn, stop etc. I believe that we will overcome this job easily, as long as we work in a planned and disciplined manner.

**Comment by yavuz-yusuf** [comment]
Hello everyone!
I am Yusuf and I am working in the embedded software team in the Hirob project. We are responsible for mechanical functionalities to Hirob. I am new in this field. Basically what i do is PCB testing and motor control.
